### ISSUE TITLE
[Bug]: Captures take too long

### ISSUE THREAD

**Issue by MarcoGonzalezM** [pokeclicker/pokeclicker#4245]
### Confirmations

- [X] I have searched open and closed issues for any existing report of my issue

### What happened?

When I throw a Ball, to a Route-wild Pokémon the capture may take near a minute to complete. This happens to me in several routes. I'm using SuperBall on the attempts in which this issue ocurr.

### What did you expect to happen?

I expected that the catch takes near a second. The Miraheze wiki reports that super ball spend 1 s on captures. (https://pokeclicker.miraheze.org/wiki/Pok%C3%A9balls=)

### Steps to Reproduce

1. Go to any route
2. Configure the catch filters to catch with Super Ball
3. Wait for a few moments until the Pokémon is defeated and the catch attempt starts
4. Sometimes it works fine, so by waiting a few more catches the bug will become visible.

### What operating system are you using?

Windows 10

### What browsers are you seeing the problem on?

Chrome

**Comment by Jaaslet** [comment]
What is a SuperBall?

**Comment by DraKay** [comment]
Great ball in some languages jaas

**Comment by MarcoGonzalezM** [comment]
Sorry, I play In Spanish so i confused the name, is Great Ball

**Comment by Jaaslet** [comment]
Did it happen before the update that went live 20 minutes ago?

**Comment by MarcoGonzalezM** [comment]
It didn't happen before

**Comment by Jaaslet** [comment]
Are you using any scripts? What browser are you using? Does it only happen for new catches, or does it happen for all catches?

**Comment by MarcoGonzalezM** [comment]
Script: gs-auto-clicker. Although it happens (but lesser times) without the auto-clicker
Browser: Chrome
I can't know if it happens for new catches, I haven't enough new catches available to test it out.

**Comment by DraKay** [comment]
After quick search on Google gs autoclicker seems like a true autoclicker not a script

**Comment by Farboo171** [comment]
Please provide a save on which this happens

**Comment by czerwinski76** [comment]
![08-05-2023_051405](https://user-images.githubusercontent.com/19463267/236726059-8106e0ed-4ee2-444c-9c22-1570a95e79be.png)

Your are lucky.
Since the latest updated, my pokeballs are gone in the space lol.

**Comment by Farboo171** [comment]
Is this bug still occurring on the latest release?

**Comment by czerwinski76** [comment]
> Is this bug still occurring on the latest release?

Yes unfortunately.

**Comment by czerwinski76** [comment]
In 2025 is this bug fixed ?

**Comment by RegisCoaxans** [comment]
No. It has been two years now and OP didn't send any file as we demanded. Moreover no similar issue has been reported since then that could point toward an actual bug. Most likely it is a user issue, not anything we can do.

**Comment by czerwinski76** [comment]
> No. It has been two years now and OP didn't send any file as we demanded. Moreover no similar issue has been reported since then that could point toward an actual bug. Most likely it is a user issue, not anything we can do.

I had that problem too.
After that, I stopped playing the game, and since I wasn't the author of the thread, I didn't share my save file at the time..

**Comment by CypherX** [comment]
The only times this issue has been observed is when using external scripts. We do not recommend or support the use of anything that modifies the game code, nor will we help when issues are encountered.

**Comment by czerwinski76** [comment]
> The only times this issue has been observed is when using external scripts. We do not recommend or support the use of anything that modifies the game code, nor will we help when issues are encountered.

I was using the Vanilla game, please.
